specs/binding-proxy.spec/src/main/resources/META-INF/zilla/proxy.idl

```diff
@@ -127,4 +127,36 @@ scope proxy
             ProxyInfo[] infos;
         }
     }
+
+    scope event
+    {
+        enum ProxyEventType (uint8)
+        {
+            REMOTE_ACCESS (1),
+            TLS (2)
+        }
+
+        enum Result (uint8)
+        {
+            SUCCESS (0),
+            FAILURE (1),
+            HANDSHAKE_TIMEOUT (2)
+        }
```

Comment:
`HANDSHAKE_TIMEOUT` is a more fine grained form of `FAILURE`, so these don't feel like they fit together.

Let's remove `Result` per other `idl` feedback and capture intent of event in the event type itself, even if we reuse the same structure for more than one event type.

Reply:
removed, see the reply above